packages/spl/src/claimable-tokens/ClaimableTokensProgram.ts

```diff
@@ -0,0 +1,313 @@
+import { struct, u8 } from '@solana/buffer-layout'
```

Comment:
why did you move away from the file organization? I kinda liked it! No problems with this either though..

Reply:
Mostly thinking about it in terms of `@audius/erc`

Reply:
I keep going back and forth on it honestly. it was a bit much to have all the prefixes, eg `createClaimableTokensAccountInstruction` vs `ClaimableTokensProgram.createAccount` or whatever. I think there's probably a way to marry the two. I took inspiration from some of Solana Web3.js spl clients, like the Secp256k1Program.